### DISCUSSION TITLE
An error occured while initializing the dbt project: No module named 'dbt.contracts.graph.parsed'

### DISCUSSION THREAD

**Discussion by brturner21** [AltimateAI/vscode-dbt-power-user#421] [Q&A] [answered]
I'm getting the above error after I updated dbt-snowflake to 1.4.2 and try opening my dbt project in vs code.  

 I'm able to use `run` and `test` within a model but `run query` and `compile` will also produce the below error:
 
` Could not execute query, because the Python bridge has not been initalized. If the issue persists, please open a Github issue.`

Any idea what's causing this?

**Comment by lf-floriandin** [comment]
Same Issue

**Comment by caseyleecfa** [comment]
Same issue here

**Comment by chenxing-mixpanel** [comment, marked as answer]
What version of dbt power user do you use? I met this issue with v0.10.1, this version somehow got cached that I can't install latest version. After I deleted and reinstalled VSCode and the extension it works.

**Reply by davidwitk** [reply]
Same issue here, somehow VS Code was only installing v0.10.1 even if a later version was available:

https://github.com/innoverio/vscode-dbt-power-user/assets/43667775/4b1daddb-8aa2-481c-a558-8d336b738c29

@chenxing-mixpanel's suggestion of uninstalling and reinstalling VS Code did the trick for me ✅

**Reply by luisvasv-teck** [reply]
The @chodera solution works for me as well

**Comment by junjiem** [comment]
Same Issue

**Comment by NoelMendoza13** [comment]
same issue

**Reply by anandgupta42** [reply]
@NoelMendoza13 Are you on the latest version of the plugin? Also, Can you share some more details?
- Operating System
- dbt version
- dbt Adapter

**Comment by gabriellegall** [comment]
same issue

**Comment by emily25inGoFreight** [comment]
same issue

**Comment by anandgupta42** [comment]
What's the version of extension are you using? @emily25inGoFreight @gabriellegall @NoelMendoza13

**Comment by emily25inGoFreight** [comment]
I just found that I was using the wrong virtual environment. I'm using v0.39.2 and it is all good now. Sorry for mistaking it.

**Comment by ShahNewazKhan** [comment]
I'm on `v0.47.4` of DBT PowerUser and still see this issue.

**Comment by fleahy-cherre** [comment]
Same here, v0.47.4.

**Comment by ShahNewazKhan** [comment]
I've been able to resolve this by ensuring I select the correct python interpreter (`ctrl-shift-p`) for my active venv with the dbt-core install.
